python/ray/tune/schedulers/pbt.py

```diff
@@ -276,13 +277,13 @@ def _log_config_on_step(self, trial_state, new_state, trial,
         ]
         # Log to global file.
         with open(os.path.join(trial.local_dir, "pbt_global.txt"), "a+") as f:
-            f.write(json.dumps(policy) + "\n")
+            f.write(json.dumps(policy, cls=_SafeFallbackEncoder) + "\n")
```

Comment:
nit:
`print(json.dumps(policy, cls=_SafeFallbackEncoder), file=f)` preferred

Reply:
oh does this automatically append a newline?

Reply:
yup!